src/app/store/selectors/ReferenceData.selectors.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { selectFeature } from '@app/store/selectors/ReferenceData.selectors';
+import { Preparer } from '@app/models/preparer';
+import { TestStation } from '@app/models/test-station';
+import { getPreparers, getTestStations } from '@app/store/selectors/ReferenceData.selectors';
+
+describe('vehicleTechRecordModel selectors', () => {
+  const createState = ({
+    referenceData = {
+      preparers: {} as Preparer[],
```

Comment:
show some intended key values here. The reader will understand what preparers and  testStations are